### ISSUE TITLE
Q&A not working . Failed to load response from ChatGPT:{"detail":{"message":"Conversation not found","code":"conversation_not_found"}}

### ISSUE THREAD

**Issue by sibillalazzerini** [sparticleinc/chatgpt-google-summary-extension#126]
**Describe the bug**
Steps Just ask anything in Q&A afrer a search.
```
Failed to load response from ChatGPT:{"detail":{"message":"Conversation not found","code":"conversation_not_found"}}
```

![Screen Shot 2023-07-01 at 10 13 42 AM](https://github.com/sparticleinc/chatgpt-google-summary-extension/assets/117636075/96b9c712-0591-4001-825b-81e32a2088eb)

**Desktop (please complete the following information):**

- OS: mac
- Browser: brave
- Version: 
![Screen Shot 2023-07-01 at 10 12 46 AM](https://github.com/sparticleinc/chatgpt-google-summary-extension/assets/117636075/305de96a-eb6f-4ffe-93de-b89857bdfcf7)

**Comment by givebest** [comment]
This issue has been fixed, but it has not yet been reviewed on the shelves, so you can try the experience: https://github.com/sparticleinc/chatgpt-google-summary-extension/tree/main/packages

**Comment by sibillalazzerini** [comment]
@givebest I see. Is this not in repo. Actually I tried cloning the repo first but it still did not work then created this issue .

**Comment by sibillalazzerini** [comment]
when was this fixed?  I am seeing this issue only for 3-4 days.
 On webstore it says last updated June 7, 2023. It's a month long thing. And many packages were updated after that, none got reviewed?

**Comment by sibillalazzerini** [comment]
Well as a fix I guess you have removed the context about the earlier answers. Now the chat thread don't remember what it said earlier. I don't think this is the kind of fix I was expecting.

**Comment by givebest** [comment]
> Well as a fix I guess you have removed the context about the earlier answers. Now the chat thread don't remember what it said earlier. isn't it ? If so then I don't think this is the kind of fix I was expecting.

Yes, this is how it is currently handled on an ad hoc basis. Keeping the same session remembering the context is the right way to go.
